docs/data/base/getting-started/quickstart/quickstart.md

```diff
@@ -38,3 +36,27 @@ Please note that [react](https://www.npmjs.com/package/react) and [react-dom](ht
   "react-dom": "^17.0.0 || ^18.0.0"
 },
 ```
+
+## Import a component
+
+```jsx
+import * as React from 'react';
+import * as NumberField from '@base_ui/react/NumberField';
+
+const NumberFieldDemo = () => (
+  <NumberField.Root>
+    <NumberField.Group>
+      <NumberField.Decrement />
+      <NumberField.Input />
+      <NumberField.Increment />
+    </NumberField.Group>
+  </NumberField.Root>
+);
+
+export default NumberFieldDemo;
```

Comment:
If we follow the convention in the docs, we would use:

```suggestion
export default function NumberFieldDemo() {
  return (
    <NumberField.Root>
      <NumberField.Group>
        <NumberField.Decrement />
        <NumberField.Input />
        <NumberField.Increment />
      </NumberField.Group>
    </NumberField.Root>
  );
}
```

the main upside is to make it easier to add hooks, not having to refactor. But since it's not a demo, same point as a above, maybe it's a case where we want to make an exception.

Reply:
Yep make an exception imo. Hooks are hugely deprioritised in new Base UI, so we should focus on simplicity in this example.